src/ontology/mondo-edit.obo

```diff
@@ -582793,6 +582792,17 @@ property_value: http://purl.org/dc/elements/1.1/date "2024-07-01T21:00:19Z" xsd:
 property_value: http://purl.org/dc/terms/creator https://orcid.org/0000-0002-0587-4693
 property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/7652" xsd:anyURI
 
+[Term]
+id: MONDO:1040032
+name: EN1-related dorsoventral syndrome
```

Comment:
@yshwetar can you please add a definition for this term?

Reply:
You can copy/paste the definition from orphanet ([here](https://www.orpha.net/en/disease/detail/611223?name=611223&mode=orpha)) and add the orphanet x-ref.

Reply:
Comment addressed